### DISCUSSION TITLE
Internal service footer

### DISCUSSION THREAD

**Discussion by martinwake** [dwp/design-system#441] [General]
Hello, we'd like to use this open discussion to talk about the [Internal service footer component](https://design-system.dwp.gov.uk/components/internal-service-footer). 

The aim of this component is to make it easier for designers to use the GOV.UK prototype kit even when designing for internal services like caseworking systems. At the moment they have to take the standard footer and remove elements such as the OGL and copyright notices and the Royal Arms: this component does that work for them.

**Comment by petejobesdigitaldwp** [comment]
When there aren't any links in the footer, do we go with just an empty grey footer to signify the end of the page still? (I'm assuming we would, but wanted to check if this scenario was considered)

**Reply by martinwake** [reply]
Hi Pete - we didn't specifically consider this scenario when making the component, but I think your assumption is a good one. However I don't know if there's a case against including empty footers? Let's consider that and possibly update in the next version of the guidance.

**Comment by jonhurrell** [comment]
@petelonsdale asked the following on slack:
>does anyone have any info regarding what links are required in the footer for an internal service?
The last time I worked on an internal service it was required that there were links to a Privacy statement and an Accessibility statement. Is this still current or have things changed?

- Added to roadmap to update guidance to cover standard links to include
- [Ticket created](https://github.com/dwp/design-system/issues/545)

**Reply by martinwake** [reply]
[General background on privacy policy links in DWP services (DWP access only)](https://dwpdigital.atlassian.net/wiki/spaces/IDNT/pages/132463296943/OIDV+privacy+notice)

**Comment by jonhurrell** [comment]
We've updated the guidance and examples, thanks @petelonsdale.
